src/pages/components/testimonial/TestimonialCard.js

```diff
@@ -11,12 +11,8 @@ const TestimonialCard = ({ item }) => {
       />
       {/* Content */}
       <figcaption className="absolute right-0 bottom-5 mx-5 pb-3 bg-gradient-to-t from-black rounded-lg text-white">
-        <p className="mb-3 mx-5 text-base font-semibold ">
-          {item.text}
-        </p>
-        <p className="mx-5 font-allison text-[2em]">
-          {item.author}
-        </p>
+        <p className="mb-3 mx-5 font-semibold">{item.text}</p>
+        <p className="mx-5 font-allison">{item.author}</p>
```

Comment:
Please return `text-[2em]` or another class to make the text bigger/clearer.

Reply:
@CharlotteDS ohhh haha forgot to put it back, I was trying something earlier